services/src/main/java/org/keycloak/truststore/TruststoreBuilder.java

```diff
@@ -85,20 +87,30 @@ static KeyStore createMergedTruststore(String[] truststores, boolean trustStoreI
             includeDefaultTruststore(truststore);
         }
 
-        mergeFiles(truststores, truststore, true);
+        List<String> loadedFiles = new ArrayList<>();
```

Comment:
Just a nitpick. I'd suggest using a more clear name, maybe `loadedFilesFromDir`?

Reply:
No problem, updated the name.